vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -542,7 +547,7 @@ function miqSetButtons(count, button_div) {
               } else {
                 tb.enableItem(button);
               }
-            } else if (onwhen = "1") {
+            } else if (onwhen == "1") {
```

Comment:
this is semantically different, does it still work?

on top of that, would not you want the '===' here? do